index.html

```diff
@@ -226,150 +249,167 @@ <h4>Multi-Post Stories Gain+Glory</h4>
+                <input class="filter" type="text" placeholder="Fullname" maxlength="30" required />
+              </fieldset>
+            </div>
 
-          <div class="div_field">
-            <fieldset>
-              <textarea class="filter" name="comment" placeholder="Enter text here" maxlength="500" required></textarea>
-            </fieldset>
-          </div>
+            <div class="div_field">
+              <fieldset>
+                <input class="filter" type="email" placeholder="Email address" required />
+              </fieldset>
+            </div>
 
-          <div class="divbtn_field">
-            <button class="btn-form btn_effects" type="submit">Get in touch</button>
-          </div>
+            <div class="div_field">
+              <fieldset>
+                <textarea
+                  class="filter txtarea_mob"
+                  name="comment"
+                  placeholder="Enter text here"
+                  maxlength="500"
+                  required
+                ></textarea>
+                <textarea
+                  class="filter txtarea_dsk"
+                  name="comment"
+                  placeholder="Typing something"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </fieldset>
+            </div>
 
-          <ul class="social_links">
-            <li>
-              <a href="https://twitter.com/marcealarconb"> <img src="images/twitter.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"><img src="images/linkedin.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"> <img src="images/medium.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"> <img src="images/github.svg" /></a>
-            </li>
-            <li>
-              <a href="#" target="_blank"> <img src="images/angellist.svg" /></a>
-            </li>
-          </ul>
-        </form>
-      </div>
-    </section>
+            <div class="divbtn_field">
+              <button class="btn-form btn_effects" type="submit">Get in touch</button>
+            </div>
+
+            <ul class="social_links">
+              <li>
+                <a href="https://twitter.com/marcealarconb"> <img src="images/twitter.svg" /></a>
+              </li>
+              <li>
+                <a href="#" target="_blank"><img src="images/linkedin.svg" /></a>
+              </li>
+              <li>
+                <a href="#" target="_blank"> <img src="images/medium.svg" /></a>
+              </li>
+              <li>
+                <a href="#" target="_blank"> <img src="images/github.svg" /></a>
+              </li>
+              <li>
+                <a href="#" target="_blank"> <img src="images/angellist.svg" /></a>
+              </li>
+            </ul>
+          </form>
+        </div>
+      </section>
+    </div>
 
     <section class="email_footer">
```

Comment:
Fantastic job @chelobotix. But kindly separate the social media section with an hr or border. And there is too much white space after the social media section. Please be strict about the Figma template and fix it. 👨‍🎨

Your design:
<img width="1122" alt="Screenshot 2022-12-16 at 12 52 36 AM" src="https://user-images.githubusercontent.com/25887253/207976384-496a31d6-33e4-4fb5-a5cb-22c61b8f2266.png">

Figma:
<img width="685" alt="Screenshot 2022-12-16 at 12 52 51 AM" src="https://user-images.githubusercontent.com/25887253/207976398-00e20fd5-36b1-4b05-b41c-0c2abd54eefd.png">

Reply:
@huseyinbiyik Thanks for r your feedback. Could you check if the overflowing issue has been resolved?
Thank you very much

Reply:
@chelobotix, they still exist. I checked on different browsers. There are still some issues.🙃